examples/advanced_example.py

```diff
@@ -0,0 +1,81 @@
+import random
+import pymilvus.orm.pymilvus_orm as orm
+from pymilvus import DataType
+from pprint import pprint
+random.seed(1234567890)
+
+
+def main():
+   
+    orm.connections.connect("default", host = "127.0.0.1", port= 19530)
+    orm.connections.connect("collection_con", host = "127.0.0.1", port= 19530)
```

Comment:
According to the import statement reviewed above, this could be simplified into

```python
connections.connect("collection_con", host="127.0.0.1", port=19530)
```
Since "default" and "collection_con" are aliases for the same server address, line10 should be removed.

Reply:
This was to show how to use alternate connections, relating to line 28. There is very little guidance on how to use different connections for Collections, Index, and Partitions, so I wanted to show the example. Its useful because when you are doing threaded inserts you dont want threads to use the same default connection.